ui/src/views/AutogenView.vue

```diff
@@ -54,7 +54,8 @@
                     v-if="!dataView && filters && filters.length > 0"
                     :placeholder="$t('label.filterby')"
                     :value="filterValue"
-                    style="min-width: 120px; margin-left: 10px; margin-top: -4px"
+                    style="min-width: 100px; margin-left: 10px; margin-top: -4px"
+                    size=small
```

Comment:
I've seen console errors on "large" and "small" in the past. Where are thos defined, and are we sure this is not an issue here?

Reply:
I dont see any errors now @DaanHoogland, probably due to ant3 upgrade.